layouts/partials/version-banner.html

```diff
@@ -20,11 +20,15 @@
 
   {{ $color := "warning" -}}
   <div class="alert alert-warning" role="alert">
-    <p>Version <strong>{{ $thisVersName }}</strong> of
+    <p>
+    <i class="fa fa-exclamation-triangle" aria-hidden="true">&nbsp;</i>
+    {{ if eq $thisVers "next" }}
```

Comment:
I like @chalin's [idea of setting `versStatus: DRAFT`](https://github.com/etcd-io/website/issues/322#issue-902574172) in `content/en/docs/next/_index.md`, and checking for that instead of checking for `"next"` here.